mhkit/wave/io/hindcast/hindcast.py

```diff
@@ -192,7 +192,11 @@ def request_wpto_point_data(
     # Construct a string representation of the function parameters
     hash_params = f"{data_type}_{parameter}_{lat_lon}_{years}_{tree}_{unscale}_{str_decode}_{hsds}_{path}_{to_pandas}"
     cache_dir = _get_cache_dir()
-    data, meta, _ = handle_caching(hash_params, cache_dir)
+    data, meta, _ = handle_caching(
+        hash_params,
+        cache_dir,
+        cache_content={"data": None, "metadata": None, "write_json": None},
+    )
```

Comment:
handle_caching reduced to only 5 inputs
https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html#max-args